mqtt/aedes/mqtt.js

```diff
@@ -109,6 +111,13 @@ esclient.on('error', function(err) {
 // MQTT over WebSocket
 function startWs() {
     var server = http.createServer();
+    server.on('request', (req, res) => {
+        if (req.url === '/version') {
+            res.statusCode = 200;
+            res.setHeader('Content-Type', 'text/plain; charset=utf-8');
+            res.end(`{"service":"mqtt-adapter","version":"${version}"}`);
+        }
```

Comment:
What if it's not version, but invalid path?

Reply:
I think that default is an empty response.